tests/lib/snaps/test-snapd-remodel-pc-24/meta/gadget.yaml

```diff
@@ -0,0 +1,57 @@
+volumes:
+  pc:
+    # bootloader configuration is shipped and managed by snapd
+    bootloader: grub
+    structure:
+      - name: mbr
+        type: mbr
+        size: 440
+        update:
+          edition: 1
+        content:
+          - image: pc-boot.img
+      - name: BIOS Boot
+        type: DA,21686148-6449-6E6F-744E-656564454649
+        size: 1M
+        offset: 1M
+        offset-write: mbr+92
```

Comment:
This is useless.

Reply:
Or even wrong probably.

Reply:
I copied from test-snapd-remodel-pc-22/meta/gadget.yaml
what should I change there?

Reply:
Remove "offset-write", "update" from the "BIOS Boot" entry.

Also in "mbr" entry, rename the image to "mbr.img" (it used to be "pc-boot.img") so that we do not confuse it if we try to use the wrong file.